src/safeposix/syscalls/fs_calls.rs

```diff
@@ -2261,12 +2261,51 @@ impl Cage {
         }
     }
 
-    //------------------------------------CHMOD HELPER FUNCTION------------------------------------
+    /// ### Description
+    ///
+    /// The `chmod_syscall()` changes a file's mode bits that consist of read, write, and execute file permission bits.
+    /// Changing set-user-ID, set-group-ID, and sticky bits is currently not supported.
+    ///
+    /// ### Arguments
+    ///
+    /// The `chmod_syscall()` accepts two arguments:
+    /// * `path` - pathname of the file whose mode bits we are willing to change (symbolic links are currently 
+    /// not supported). If the pathname is relative, then it is interpreted relative to the current working directory 
+    /// of the calling process.
+    /// * `mode` - the new file mode, which is a bit mask created by bitwise-or'ing zero or more valid mode bits.
+    /// Some of the examples of such bits are `S_IRUSR` (read by owner), `S_IWUSR` (write by owner), 
+    /// `S_ISUID` (set-user-ID), etc.
```

Comment:
Do we really have S_ISUID?

Reply:
this is a general description of the syscall based on the standard. before that part, right in the beginning, I do mention that changing set-user-ID, set-group-ID, and sticky bits is not supported in the current implementation.